js/repl/src/index.css

```diff
@@ -0,0 +1,24 @@
+html,
+body {
+  height: 100%;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
```

Comment:
I guess this body styling can be removed once we merge the REPL into the site itself?